src/main/java/cn/nukkit/block/BlockSugarcane.java

```diff
@@ -104,8 +104,11 @@ public int onUpdate(int type) {
                     for (int y = 1; y < 3; ++y) {
                         Block b = this.getLevel().getBlock(new Vector3(this.x, this.y + y, this.z));
                         if (b.getId() == AIR) {
-                            this.getLevel().setBlock(b, Block.get(BlockID.SUGARCANE_BLOCK), false);
-                            break;
```

Comment:
Is there any reason why you removed the break statement?

Reply:
> 
> 
> Is there any reason why you removed the break statement?

Sugar canes grow 2 blocks at the same time in minecraft.